README.md

```diff
@@ -161,6 +161,7 @@ Compared to ChatGLM's [P-Tuning](https://github.com/THUDM/ChatGLM2-6B/tree/main/
 | [Falcon](https://huggingface.co/tiiuae)                      | 7B/11B/40B/180B                  | falcon    |
 | [Gemma/Gemma 2/CodeGemma](https://huggingface.co/google)     | 2B/7B/9B/27B                     | gemma     |
 | [GLM-4](https://huggingface.co/THUDM)                        | 9B                               | glm4      |
+| [GLM-4V](https://huggingface.co/THUDM)                       | 13B(include vision)              | glm4_v    |
```

Comment:
No need for "include vision"
I think it is better to use 9B as the official report

Reply:
But while using vision it will load all 13b params,I think it will lead to some misunderstanding and confuse about vram usage?

Reply:
I think it is all right, as `7b` in `llava-7b` also ignore the vision module param size, only takes the 7b in `vicuna-7b`, how do you think?

Reply:
I think 9B in GLM-4V 9B, also want to notify user, their `text decoder` is GLM-9B rather than 13B

Reply:
Ok i'll use it